nucleus/generate.py

```diff
@@ -163,18 +161,19 @@ def validate_config(config: dict):
 
     if "gpu_version" not in config:
         config["gpu_version"] = None
-    if "tfs_gpu" not in config:
-        config["tfs_gpu"] = None
-    if config["gpu_version"] and config["tfs_gpu"]:
-        raise CortexModelServerBuilder("can only specify 'gpu_version' or 'tfs_gpu'")
+    if "gpu" not in config:
+        config["gpu"] = False
 
-    if config["gpu_version"] and config["type"] == "tensorflow":
+    if config["type"] == "python" and (
+        (not config["gpu"] and config["gpu_version"])
```

Comment:
this should result in a different error message.

e.g. "gpu must be enabled (by setting 'gpu: true') in order to specify gpu_version"